internal/cli/atlas/livemigration/live_migration.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package livemigration
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/atlas/livemigration/link"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "liveMigrations"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: cli.GenerateAliases(use, "lm"),
+		Short:   "Manage Atlas Live Migration for your org.",
+		Long:    "The liveMigrations command provides access to your org live migrations. You can create, edit, and delete live migrations.",
```

Comment:
Nit: Can you actually edit a live migration? I don't think I see that in the API. A wording suggestion that might future-proof us if we add/change functionality:
```suggestion
		Long:    "Manage a Live Migration to Atlas for your organization.",
```